openquakeplatform/openquakeplatform/exposure/static/exposure/exposure_controller.js

```diff
@@ -279,18 +313,18 @@ app.controller('ExposureRegionList', function($scope, $filter, $http, myService,
     // Sub-national level building exposure download form
     function subNationalForm(study) {
         return '<form id="sub-exposure-building-form" class="exposure_export_form">'+
-                '<p><b>Download Region Wide Building/Dwelling Fractions:</b></p><button id="dwellingFractionsDownload" type="button" value="'+study.study_region_id+'">Download</button></br></br>'+
-                '<b>Download Gridded Sub-National Building Exposure:</b></br>'+
-                '<p><label for="id_residential_0">Building Type:</label></br>'+
-                '<label for="id_residential_0"><input class="exposure_export_widget" id="id_residential_0" name="sub-residential" type="radio" checked="" value="residential" /> Residential</label></br>'+
-                '<label id="id_residential_1_text" for="id_residential_1"><input class="exposure_export_widget" id="id_residential_1" name="sub-residential" type="radio" value="non-residential" /> Non-Residential</label></br>'+
-                '</p>'+
-                '<p><label for="id_outputType_0">Output Type:</label></br>'+
-                '<label for="id_outputType_0"><input class="exposure_export_widget" id="id_outputType_0" name="sub-outputType" type="radio" checked="" value="csv" /> CSV</label></br>'+
-                '<label for="id_outputType_1"><input class="exposure_export_widget" id="id_outputType_1" name="sub-outputType" type="radio" value="nrml" /> NRML</label></br>'+
-                '</p>'+
-                '<input type="hidden" name="sub-study" value="'+study.study_region_id+'">'+
-                '<br>'+
+            '<p><b>Download Region Wide Building/Dwelling Fractions:</b></p><button id="dwellingFractionsDownload" type="button" value="'+study.study_region_id+'">Download</button></br></br>'+
+            '<b>Download Gridded Sub-National Building Exposure:</b></br>'+
```

Comment:
Could we move all this HTML code to the html page/template and hide it with a `display: none` css directive ?
Before show it we can set the `sub-study` input tag with a proper jquery call.